src/components/originalSampleProcessing/ProcessingSuccess.tsx

```diff
@@ -34,7 +35,11 @@ const ProcessingSuccess: React.FC<ProcessingSuccessProps> = ({
           variants={variants.fadeInWithLift}
           className="sm:max-w-xl w-full border-gray-200 p-4 rounded-md bg-gray-100 shadow"
         >
-          <LabelPrinter labwares={labware} />
+          <LabelPrinter
+            labwares={labware.filter(
+              (lw) => lw.labwareType.name !== LabwareTypeName.PRE_BARCODED_TUBE
```

Comment:
Would it make more sense to check if the labware type has a _labelType_ instead of checking its name?

Reply:
Pushed in changes.